linter/lintmain/internal/check/check.go

```diff
@@ -246,6 +249,29 @@ func (l *linter) loadPlugin() error {
 }
 
 func (l *linter) parseArgs() error {
+	paramKey := func(info *lintpack.CheckerInfo, paramName string) string {
+		return "@" + info.Name + "." + paramName
+	}
+
+	intParams := make(map[string]*int)
+	boolParams := make(map[string]*bool)
+	stringParams := make(map[string]*string)
+	for _, info := range l.infoList {
```

Comment:
What about extracting this and following loops to a func?

Reply:
Done.

Reply:
Nice, LGTM